### PR TITLE
Added configuration for Jetty thread pool and queue size

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: etc/conf.templates/distributed/10-egress.conf.template [hunk 1]
```diff
@@ -48,6 +48,16 @@ egress.port = 8881
 //
 egress.host = 127.0.0.1
 
+//
+// Number of threads in Jetty's Thread Pool
```
Comment (ftence) [inline etc/conf.templates/distributed/10-egress.conf.template:52]: Defaults to 200.

File: etc/conf.templates/standalone/00-warp.conf.template [hunk 2]
```diff
@@ -92,6 +92,16 @@ standalone.host = 127.0.0.1
 //
 standalone.port = 8080
 
+//
+// Number of threads in Jetty's Thread Pool
```
Comment (ftence) [inline etc/conf.templates/standalone/00-warp.conf.template:96]: Defaults to 200.

File: warp10/src/main/java/io/warp10/continuum/egress/Egress.java [hunk 3]
```diff
@@ -103,6 +106,16 @@ public Egress(KeyStore keystore, Properties props, boolean fetcher) throws Excep
...
     
+    int maxThreads = Integer.parseInt(props.getProperty(Configuration.EGRESS_JETTY_THREADPOOL, DEFAULT_THREADPOOL_SIZE));
+    BlockingArrayQueue<Runnable> queue = null;
+    
+    if (props.containsKey(Configuration.EGRESS_JETTY_MAXQUEUESIZE)) {
+      int queuesize = Integer.parseInt(props.getProperty(Configuration.EGRESS_JETTY_MAXQUEUESIZE));
+      queue = new BlockingArrayQueue<Runnable>(queuesize);
+    }
+    
+    Server server = new Server(new QueuedThreadPool(maxThreads, 8, 60000, queue));
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/continuum/egress/Egress.java:117]: Remove `Server` at the start of the line and the following line.

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 4]
```diff
@@ -72,6 +74,8 @@
 
 public class Warp extends WarpDist implements Runnable {
 
+  private static final String STANDALONE_THREADPOOL_SIZE = "200";
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:77]: Add `DEFAULT` to the name to make it clear it is a default value.

**Review by ftence — CHANGES_REQUESTED** [review]
There is also PlasmaFrontEnd to fix.

**Comment by hbs** [comment]
PTAL

**Review by ftence — APPROVED** [review]

**Pull request merged**